.changeset/upset-paths-stand.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/container-loader": major
+---
+
+Loader container caching is removed
```

Comment:
It isn't clear from this and the description what is not cached anymore. Maybe say something like - Container caching in the Loader has been removed.